scene/main/viewport.cpp

```diff
@@ -162,6 +162,24 @@ ViewportTexture::~ViewportTexture() {
 }
 
 void Viewport::_sub_window_update_order() {
+	if (gui.sub_windows.size() < 2) {
+		return;
+	}
+
+	int index = gui.sub_windows.size() - 1;
+	if (!gui.sub_windows[index].window->get_flag(Window::FLAG_ALWAYS_ON_TOP)) {
+		SubWindow sw = gui.sub_windows[index];
```

Comment:
`sw` is only used within the if-block below. You could move the declaration down there.

Reply:
alright, also moved `index` initialization inside the if